spec/features/govbox_registration_spec.rb

```diff
@@ -35,6 +35,43 @@
     expect(page).to have_content('Registrácia úspešná!')
   end
 
+  scenario 'User registers for govbox API' do
+    visit '/sluzby/govbox'
+    within('#registration') do
+      click_on 'Mám záujem o GovBox'
+    end
+
+    fill_in 'Názov', with: 'Slovensko.Digital'
+    fill_in 'IČO', with: '50 158 635'
+
+    click_on 'Ďalej'
+
+    fill_in 'Meno', with: 'Ján'
+    fill_in 'Priezvisko', with: 'Hargaš'
+    fill_in 'Trvalé bydlisko', with: 'Koprivnická 9/B, 841 04 Bratislava'
+    click_on 'Ďalej'
+
+    fill_in 'Email', with: 'jan.hargas@slovensko.digital'
+    fill_in 'Mobilný telefón', with: '+421903919123'
+    check('mode')
+    click_on 'Ďalej'
+
+    fill_in 'Zvoľte si heslo', with: 'nejakeheslo'
+    fill_in 'Heslo (znova)', with: 'nejakeheslo'
+
+    WebMock.allow_net_connect!
```

Comment:
a toto robi co? a preco to tu je?

Reply:
To som sa chcel pytat, bez toho mi pada specs s dakou cudnou chybou (poslem log hned). na stackoverflow to davali ako solution

Reply:
@pavolzbell nevies co to je?

![image](https://user-images.githubusercontent.com/44023416/65632652-7af12780-dfda-11e9-8043-ed04ba50c738.png)

Reply:
@smith558 kedze ten controller postne request po registracii na govbox endpoint tak toto to dovoli urobit. Cize to tam nechajte.

Reply:
@jsuchal a preco rovnako nepada aj ten priklad pre registraciu bez API mode?

Reply:
@pavolzbell som to pustal teraz, pada aj bez API mode. Iba ked ma spec len 1 request, ked su 2 a jeden bez API tak prechadza. @jsuchal necham to tak teda

Reply:
> @jsuchal a preco rovnako nepada aj ten priklad pre registraciu bez API mode?

@jsuchal okej necham tak. ale tiez ma zaujima 👆

Reply:
@smith558 kukam, ze si to nakoniec vyrazil https://github.com/slovensko-digital/www.ecosystem/pull/40/commits/62421db330d43baf66bfa59be4f718d69324a5c5 v com bol teda problem?

Reply:
@pavolzbell spec failoval spravne, mal som tam chybu v params implementacii, ten popisok len nebol moc informacny k chybe

oprava -> https://github.com/slovensko-digital/www.ecosystem/commit/180e2c1666021ebc58a51bf85d31c91ffd26a89f